resources/js/Shared/Select.vue

```diff
@@ -21,25 +24,14 @@
         {{ $t('app.optional') }}
       </span>
     </label>
-    <v-select v-model="selected"
-              :options="options"
-              :placeholder="placeholder"
-              class="style-chooser"
-              :label="customLabelKey"
-              :data-cy="datacy"
-              :close-on-select="true"
-              @input="broadcast"
-    >
-      <!-- all this complex code below just to make sure the select box is required -->
-      <template #search="{attributes, events}">
-        <input
-          class="vs__search"
-          :required="required && !selected"
-          v-bind="attributes"
-          v-on="events"
-        />
-      </template>
-    </v-select>
+    <vue-select v-model="proxyValue"
```

Comment:
I dont think we should do that.
Either we get rid of the v-select component, or we simply try to do our own.

Reply:
For now it's the best solution I had

Reply:
vue-select component is back again!